M/MParT/build_tarballs.jl

```diff
@@ -2,16 +2,27 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
+# See https://github.com/JuliaLang/Pkg.jl/issues/2942
+# Once this Pkg issue is resolved, this must be removed
+uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
+julia_versions = [v"1.6.3", v"1.7.0", v"1.8.0", v"1.9.0"]
 name = "MParT"
-version = v"0.1.0"
+version = v"0.2.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/MeasureTransport/MParT.git", "64e4b8ee7637382ab7f9aa091ace41ed6a5cbcc6")
+    GitSource("https://github.com/MeasureTransport/MParT.git", "7e138bc9981f4181db0388a2a481155a45f0eafe")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+# Override compiler ID to silence the horrible "No features found" cmake error
+if [[ $target == *"apple-darwin"* ]]; then
+  macos_extra_flags="-DCMAKE_CXX_COMPILER_ID=AppleClang -DCMAKE_CXX_COMPILER_VERSION=10.0.0 -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=11"
```

Comment:
I don't know what horrible error you're talking about, but this sounds like a lie since we do _not_ use AppleClang

Reply:
Sorry if this is wrong, but there are a few examples using cxxwrap on Yggdrasil that use this exact thing (jlqml, sdpa, spot_julia), and I wasn't sure if I needed to include it so i erred on the side of caution. I'll double check if it's needed ASAP.

Reply:
looks like it isn't needed, at least according to Azure.